docs/code_conventions/README.MD

```diff
@@ -34,6 +34,41 @@ apiVersion = "v1"
 username = demisto.params().get("url")
 ```
 
+## Arguments
```

Comment:
But all caps variable in Python is a const variable by definition, if you try to change it in one of your functions it will get an exception

Reply:
@anara123 @ronykoz  What do we want to do here? I think we are confusing Python syntax with quasi-Demisto branded language. I think Rony is correct here, but we as a company have been referring to these as Global Variables (since within Demisto's UI they are variable) for quite some time.

I see two solutions here:
1. Continue to use Global Variables
2. Introduce Something like "Integration Constants" along side our use of "Global Variables" and slowly phase out "Global Variables"

I like 2 since it moves us towards language that is more indicative of what it does, and introducing it slowly alongside Global Variables will help customers and users make the connection that they are the same.

@kirbles19 @yuvalbenshalom  - What are your thoughts?

Reply:
I'm not sure that customers are familiar with these fields as Global variables if they are familiar with them at all. So in my opinion there is no need to phase out, just rename them internally and introduce them to the customers with it's new name for the first time
We can call them `Instance Constants` if you like because they are per instance of an integration and not shared between different instances.

Reply:
it is not only instance constants 
there are times you define constants
for example:
`DAY_IN_MILLISECONDS = 24 * 60 * 60 * 1000`
I think `Global/Constant Variables` should be fine